docs/use-the-network/run-a-network-server/buy-an-oui/buy-an-oui.mdx

```diff
@@ -0,0 +1,124 @@
+| 12yUUZmsVyVaomxkvuNXtD2rAHYWgYwJLXtz4H9FjYgE2WcniLx | 0.00000000 | 0            | 0.00000000      |
++-----------------------------------------------------+------------+--------------+-----------------+
+```
+
+If we don't have an account balance of at least 90000000 DCs ($900), we need to
+create the data credits now.
+
+Inspect
+[the current oracle price](https://api.helium.io/v1/oracle/prices/current) and
+you'll get the current price of HNT as agreed by the Helium Network; it's
+represented in 10^-8 dollars. Suppose it is currently $4.5, so we would get:
+
+```text
+{"data":{"timestamp":"2021-02-25T10:01:32.000000Z","price":450000000,"block":735970}}
+```
+
+Since 1 DC = $.00001 = $10^-5 and we need to normalize with the $10^-8 from the
+oracle price, we can plug the following into our favorite calculate (Python for
+me):
+
+```text
+>>> 90000000*10**3/450000000
+200.0
+```
+
+I need to **200 HNT** to get $900 of DC (plus a little extra for the OUI txn
+fee). From _any_ wallet with ~201 HNT, I need to initiate the following
+transaction, which sets the wallet I just created as the beneficiary of the
+burn:
+
+```text
+./helium-wallet burn --amount 200.5 --payee 12yUUZmsVyVaomxkvuNXtD2rAHYWgYwJLXtz4H9FjYgE2WcniLx --commit
+```
+
+This submits the transaction and the CLI will output the transaction hash which
+you can track at the following endpoint:
+
+```
+https://api.helium.io/v1/pending_transactions/HASH
+```
+
+When the transaction clears, our balance should look like this:
+
+```text
+$ ./helium-wallet balance
++-----------------------------------------------------+------------+--------------+-----------------+
+| Address                                             | Balance    | Data Credits | Security Tokens |
++-----------------------------------------------------+------------+--------------+-----------------+
+| 12yUUZmsVyVaomxkvuNXtD2rAHYWgYwJLXtz4H9FjYgE2WcniLx | 0.00000000 | 90225000     | 0.00000000      |
++-----------------------------------------------------+------------+--------------+-----------------+
+```
+
+## Purchase the OUI
+
+The next part is simple. Submit a "create OUI" transaction:
+
+```text
+./helium-wallet oui create --subnet-size 8 --filter wVwCiewtCpEKAAAAAAAAAAAAcCK3fwAAAAAAAAAAAABI7IQOAHAAAAAAAAAAAAAAAQAAADBlAAAAAAAAAAAAADEAAAA2AAAAOgAAAA
+```
+
+The filter is a dummy filter to initialize the OUI. When you get Console
+running, it will automatically maintain it. You can once again track the
+transaction with the endpoint above. When it is complete, you should also see
+your OUI as the [most recently purchased](https://api.helium.io/v1/ouis/last)
+and on
+[this list here](https://api.helium.io/v1/ouis?cursor=eyJ0eXBlcyI6Im91aV92MSIsImJsb2NrIjo3MDUxMDAsImFuY2hvcl9ibG9jayI6NzA1MTAwf).
+
+Congratulations! You are the proud owner of a Helium Network OUI. OUIs are
+numbered sequentially, so you get street cred for small numbers.
```

Comment:
```suggestion
numbered sequentially, so the lower you are, the earlier you were on the Network!.
```

Reply:
Shame. I thought you'd enjoy discussions of street cred

Reply:
I'm a fan, but I'm not so sure about non-native English speakers.

Reply:
yea I agree.